cmake/ClangToolsLibsettings.cmake

```diff
@@ -49,15 +49,15 @@ endif()
 
 if (EXISTS ${CLANG_FORMAT_PATH})
     # Format all files .c files (and their headers) in project
-    add_custom_target(clang-format-all
+    add_custom_target(clang-format-all-libsettings
```

Comment:
this seems a bit awkward, I wonder if there is a way to refactor the `.cmake` module so that it can be reused without these sort of naming gymnastics